opennlp-dl/pom.xml

```diff
@@ -38,7 +38,7 @@
     </dependency>
     <dependency>
       <groupId>com.microsoft.onnxruntime</groupId>
-      <artifactId>onnxruntime</artifactId>
+      <artifactId>onnxruntime_gpu</artifactId>
```

Comment:
This dependency works for both CPU and GPU.

Reply:
:+1: if you think it's helpful maybe this note can be added as a comment to the `pom.xml`.